### PR TITLE
fix(Slider): provide LabelText for TextInput from Slider or hide it

### LINKED ISSUE #227: Need a fix to update LabelText for TextInput in default Slider or hide it
TextInput Label is not available to Slider

### PR DESCRIPTION
**Pull request by AsthaJain1**
- Fix Slider to be capable of providing labelText for TextInput or hide it
- Closes #227

### CODE REVIEW

File: .storybook/components/SliderStory.js [hunk 1]
```diff
@@ -12,7 +12,18 @@ storiesOf('Slider', module)
...
       <div style={{ marginTop: '2rem' }}>
-        <Slider id="slider" value={50} min={0} max={100} step={1} labelText="Slider Label" onChange={mock} />
+        <Slider id="slider" value={50} min={0} max={100} step={1} labelText="Slider Label" onChange={mock} labelForTextInput="Enter value"  />
+      </div>
+    )
+  )
+    .addWithInfo(
+    'hide Label',
+    `
+      Sliders provide a visual indication of adjustable content, where the user can move the handle along a horizontal track to increase or decrease the value.
```
Comment (jlpadilla) [inline .storybook/components/SliderStory.js]: Update this description.  Maybe something like:
> Label can be hidden using the `hideTextInputLabel` property.

**Comment by chrisdhanaraj** [comment]
I think it actually probably makes more sense to just remove the defaultProp on TextInput, rather than adjust the API here

**Comment by AsthaJain1** [comment]
https://github.com/carbon-design-system/carbon-components-react/pull/153
As per this PR - removing the defaultProp on TextInput will have impact on accessibility.

**Comment by chrisdhanaraj** [comment]
Brian's PRs there had some misconceptions about the i18n requirements - the only strings that needed to be defaulted were required strings. The label was a conditional string, so shouldn't have a defaultProp

**Comment by tw15egan** [comment]
@AsthaJain1 could you update this PR to just remove the defaultProp on TextInput?

**Comment by AsthaJain1** [comment]
@tw15egan i'll put another pr.

**Comment by tw15egan** [comment]
@AsthaJain1 thank you!

**Comment by AsthaJain1** [comment]
@tw15egan Hey, finally found the repo. I'll use the same PR. sorry about that. Thank you for not closing this PR already!

**Comment by tw15egan** [comment]
Great! Thanks for taking care of this! 👍 ✅

**Comment by chrisdhanaraj** [comment]
Bluemix [toolchain](https://console.bluemix.net/devops/toolchains/6b8b7163-1ec8-4616-82fa-aa467a17af03 "storybook-toolchain-v2"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/b84aaac8-8484-4a1e-902a-ebb526eb7998 "storybook-toolchain-v2") deployed [carbon-components-react](https://github.com/carbon-design-system/carbon-components-react/releases/tag/deploy-ibm-yp-us-south-chrisdhanaraj_org-Bluemix-Design-System-20171011-005532 "deploy-ibm-yp-us-south-chrisdhanaraj_org-Bluemix-Design-System-20171011-005532") to [Bluemix Design System](https://console.bluemix.net/apps/bc75a101-6472-481c-b48d-3eed1e9703f3?env_id=ibm:yp:us-south "ibm:yp:us-south:chrisdhanaraj_org:Bluemix Design System"), including commit 46b39a578ed2a650343ca5d2baaec9407df1fd24

**Pull request merged**